src/test/java/io/opentracing/contrib/global/concurrent/AddSuppressedSupportTest.java

```diff
@@ -0,0 +1,25 @@
+package io.opentracing.contrib.global.concurrent;
+
+import org.junit.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.Matchers.sameInstance;
+
+/**
+ * @author Sjoerd Talsma
+ */
+public class AddSuppressedSupportTest {
+
+    @Test
+    public void testExceptionCombinations() {
+        final Exception ex1 = new Exception(), ex2 = new Exception();
+        assertThat(AddSuppressedSupport.addSuppressedOrLog(null, null, null), is(nullValue()));
+        assertThat(AddSuppressedSupport.addSuppressedOrLog(ex1, null, "message"), is(sameInstance(ex1)));
```

Comment:
this test doesn't help me understand why we have the utility. if we keep code like this (which I don't think we should) it should be explained in a feature test.